tools/generate-classes/index.js

```diff
@@ -77,6 +79,16 @@ for (const schemaPath of argv.schemas) {
 
 const schemaCache = new SchemaCache(schemaBasePaths, argv.extensions);
 
+// Add extensions to schema cache so that extensions can extend objects in other extensions
+for (const extension of config.extensions) {
+  const extensionSchema = schemaCache.loadExtension(
+    extension.schema,
+    extension.extensionName
+  );
+  const { _, schemaBasePath } = splitSchemaPath(extensionSchema.sourcePath);
+  schemaCache.schemaPaths.push(schemaBasePath);
```

Comment:
I don't really like how this is modifying `schemaPaths`, but this approach didn't require too many code changes.

Reply:
@kring Do you have any concerns with this change? If not, happy to merge this. Just wanted to double-check

Reply:
No concerns, it seems reasonable enough to me.